src/Handler/Mapping/MethodName/Handle.php

```diff
@@ -7,7 +7,7 @@
 /**
  * Handle command by calling the "handle" method.
  */
-class Handle implements MethodNameInflector
+final class Handle implements MethodNameInflector
 {
     /**
      * {@inheritdoc}
```

Comment:
Also groovy! I should just setup my own coding standards thing, similar to doctrine-coding-standards